fortigate-autoscale/fortigate-bootstrap-config-strategy.ts

```diff
@@ -336,6 +353,16 @@ export class FortiGateBootstrapConfigStrategy implements BootstrapConfigurationS
             return config;
         }
     }
+
+    /**
+     * To normalize a string in order for a safe use as the input value for the FOS commands.
+     *
+     * @param {string} input the input string to normalize
+     * @returns {string} the normalized input string
+     */
+    protected normalizeFOSCmdInput(input: string): string {
+        return (input && input.replace(/\\/g, '\\\\').replace(/"/g, '\\"')) || null;
```

Comment:
there's probably other special characters?

I usually do something like
```
input.replace(/\\"'/g, m => `\\${m}`)
```

https://help.fortinet.com/fweb/4-3-5/cli/wwhelp/wwhimpl/common/html/wwhelp.htm?context=FortiWeb_Online_CLI_Reference&file=CLI_basics_3_15.html

Reply:
this link is very helpful. I'll update the code.